server/app/views/components/ProgramQuestionBank.java

```diff
@@ -145,25 +148,49 @@ private FormTag questionBankPanel() {
                                     params.questionCreateRedirectUrl(),
                                     /* isPrimaryButton= */ false)))));
 
-    ImmutableList<QuestionDefinition> questions =
+    // Sort by last modified, since that's the default of the sort by dropdown
+    ImmutableList<QuestionDefinition> allQuestions =
         filterQuestions()
             .sorted(
                 Comparator.<QuestionDefinition, Instant>comparing(
                         qdef -> qdef.getLastModifiedTime().orElse(Instant.EPOCH))
                     .reversed()
                     .thenComparing(qdef -> qdef.getName().toLowerCase(Locale.ROOT)))
             .collect(ImmutableList.toImmutableList());
-
+    ImmutableList<QuestionDefinition> universalQuestions =
+        allQuestions.stream().filter(q -> q.isUniversal()).collect(ImmutableList.toImmutableList());
+    ImmutableList<QuestionDefinition> nonUniversalQuestions =
+        allQuestions.stream()
+            .filter(q -> !q.isUniversal())
+            .collect(ImmutableList.toImmutableList());
+    if (!universalQuestions.isEmpty() && showUniversal) {
+      contentDiv.with(
```

Comment:
suggestion (non-blocking):  The creation of the question bank subsection div could be separated into a helper function to clean things up and for a little bit less duplication.